src/pages/Dashboard/data.ts

```diff
@@ -1,137 +1,57 @@
-// Below is dummy data for the default countries that are always rendered.
-// They are now in back-end format
-export const data = [
-  {
-    id: 9999,
-    name: "Afghanistan",
-    continent: "Asien",
-    last_modified: "2023-05-14T12:47:24.455403",
-    total: 4000,
-    lgbt: 300,
-    status: {
-      Accepted: 1500,
-      Rejected: 2500,
-      Unknown: 100,
-    },
-    gender: {
-      Male: 100,
-      Female: 60,
-      Other: 25,
-      Unknown: 5,
-    },
-  },
-  {
-    id: 99999,
-    name: "Belize",
-    continent: "Sydamerika",
-    last_modified: "2023-05-14T12:47:24.455403",
-    total: 3000,
-    lgbt: 100,
-    status: {
-      Accepted: 500,
-      Rejected: 2500,
-      Unknown: 100,
-    },
-    gender: {
-      Male: 75,
-      Female: 50,
-      Other: 50,
-      Unknown: 5,
-    },
-  },
-  {
-    id: 999999,
-    name: "Cambodia",
-    continent: "Asien",
-    last_modified: "2023-05-14T12:47:24.455403",
-    total: 3000,
-    lgbt: 180,
-    status: {
-      Accepted: 500,
-      Rejected: 2500,
-      Unknown: 100,
-    },
-    gender: {
-      Male: 35,
-      Female: 20,
-      Other: 30,
-      Unknown: 5,
-    },
-  },
-  {
-    id: 9999999,
-    name: "Djibouti",
-    continent: "Afrika",
-    last_modified: "2023-05-14T12:47:24.455403",
-    total: 2780,
-    lgbt: 200,
-    status: {
-      Accepted: 1000,
-      Rejected: 1780,
-      Unknown: 100,
-    },
-    gender: {
-      Male: 60,
-      Female: 10,
-      Other: 65,
-      Unknown: 1,
-    },
-  },
-  {
-    id: 99999999,
-    name: "Ecuador",
-    continent: "Sydamerika",
-    last_modified: "2023-05-14T12:47:24.455403",
-    total: 4800,
-    lgbt: 400,
-    status: {
-      Accepted: 2600,
-      Rejected: 2200,
-      Unknown: 100,
-    },
-    gender: {
-      Male: 30,
-      Female: 40,
-      Other: 170,
-      Unknown: 5,
-    },
-  },
-  {
-    id: 999999999,
-    name: "Finland",
-    continent: "Europa",
-    last_modified: "2023-05-14T12:47:24.455403",
-    total: 500,
-    lgbt: 400,
-    status: {
-      Accepted: 90,
-      Rejected: 410,
-      Unknown: 100,
-    },
-    gender: {
-      Male: 50,
-      Female: 50,
-      Other: 25,
-      Unknown: 10,
-    },
-  },
-  {
-    id: 9999999999,
-    name: "Ghana",
-    continent: "Afrika",
-    last_modified: "2023-05-14T12:47:24.455403",
-    total: 3490,
-    lgbt: 310,
-    status: {
-      Accepted: 1300,
-      Rejected: 2190,
-      Unknown: 100,
-    },
-    gender: {
-      Male: 90,
-      Female: 120,
-      Other: 25,
-      Unknown: 5,
-    },
-  },
-];
+import { useEffect, useState } from 'react';
+
+// Definition of country object type
+interface Country {
```

Comment:
is this already defined somewhere else? if so we should import that definition to avoid multiple versions of what are meant to be the same type
if we have multiple then we might run into issues down the line if we change the definition of one and forget to change the definition of the other